challenges/ fizzBuzzTree/__tests__/fizz-buzz.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const treeTester = require('../fizz-buzz-tree');
+
+
+
+// Breadth Test
+describe('FizzBuzz Tests', () => {
+  it('Successfully returns a new tree without mutating the original', () => {
+
+/*     const tree = new treeTester.BinaryTree();
+
+    tree.root = new treeTester.Node(5);
+    tree.root.left = new treeTester.Node(10);
+    tree.root.left.left = new treeTester.Node(0);
+    tree.root.left.right = new treeTester.Node(20);
+    tree.root.right = new treeTester.Node(15);
+    tree.root.right.right = new treeTester.Node(25);
+
+    const testValue = tree.breadthFirst();
+
+    expect(testValue).toEqual([ 5, 10, 15, 0, 20, 25 ]); */
+  });
```

Comment:
I would like to see tests and a README added to this and then I will regrade for you!